diff --git a/proxy/src/db/schema.rs b/proxy/src/db/schema.rs
index 5a66ac4..81c405a 100644
--- a/proxy/src/db/schema.rs
+++ b/proxy/src/db/schema.rs
@@ -1,43 +1,43 @@
 table! {
     bookmarks (id) {
-        id -> Integer,
+        id -> BigInt,
         name -> Nullable<Text>,
         address -> Text,
-        channel -> Nullable<Integer>,
-        identity -> Integer,
+        channel -> Nullable<BigInt>,
+        identity -> BigInt,
         bookmark -> Bool,
         last_used -> Nullable<Timestamp>,
-        server -> Nullable<Integer>,
+        server -> Nullable<BigInt>,
     }
 }
 
 table! {
     channel_messages (server, channel, message) {
-        server -> Integer,
-        channel -> Integer,
-        message -> Integer,
+        server -> BigInt,
+        channel -> BigInt,
+        message -> BigInt,
     }
 }
 
 table! {
     channels (server, id) {
-        server -> Integer,
-        id -> Integer,
-        parent -> Nullable<Integer>,
+        server -> BigInt,
+        id -> BigInt,
+        parent -> Nullable<BigInt>,
         name -> Text,
         icon -> Nullable<Integer>,
         deleted -> Bool,
     }
 }
 
 table! {
     client_messages (server, client, message) {
-        server -> Integer,
+        server -> BigInt,
         client -> Binary,
-        message -> Integer,
+        message -> BigInt,
     }
 }
 
 table! {
     clients (uid) {
         uid -> Binary,
@@ -47,63 +47,63 @@ table! {
         custom_name -> Nullable<Text>,
     }
 }
 
 table! {
     events (id) {
-        id -> Integer,
-        server -> Nullable<Integer>,
+        id -> BigInt,
+        server -> Nullable<BigInt>,
         invoker -> Nullable<Binary>,
-        channel1 -> Integer,
-        channel2 -> Integer,
+        channel1 -> BigInt,
+        channel2 -> BigInt,
         client -> Nullable<Binary>,
         typ -> Text,
         content -> Nullable<Binary>,
         time -> Timestamp,
     }
 }
 
 table! {
     identities (id) {
-        id -> Integer,
+        id -> BigInt,
         private_key -> Binary,
         name -> Text,
-        offset -> Integer,
-        max_counter -> Integer,
+        offset -> BigInt,
+        max_counter -> BigInt,
         client -> Binary,
     }
 }
 
 table! {
     messages (id) {
-        id -> Integer,
+        id -> BigInt,
         invoker -> Nullable<Binary>,
         content -> Text,
         time -> Timestamp,
     }
 }
 
 table! {
     server_messages (server, message) {
-        server -> Integer,
-        message -> Integer,
+        server -> BigInt,
+        message -> BigInt,
     }
 }
 
 table! {
     servers (id) {
-        id -> Integer,
+        id -> BigInt,
         name -> Text,
         address -> Text,
         icon -> Nullable<Integer>,
     }
 }
 
 table! {
     servers_clients (server, client) {
-        server -> Integer,
+        server -> BigInt,
         client -> Binary,
         last_seen -> Timestamp,
     }
 }
 
 joinable!(bookmarks -> identities (identity));
