diff --git a/proxy/src/db/schema.rs b/proxy/src/db/schema.rs
index 46c2103..319f34e 100644
--- a/proxy/src/db/schema.rs
+++ b/proxy/src/db/schema.rs
@@ -1,10 +1,10 @@
 table! {
     bookmarks (id) {
-        id -> Integer,
+        id -> BigInt,
         name -> Nullable<Text>,
         address -> Text,
-        channel -> Nullable<Integer>,
-        identity -> Integer,
+        channel -> Nullable<BigInt>,
+        identity -> BigInt,
         bookmark -> Bool,
         last_used -> Nullable<Timestamp>,
         server -> Nullable<Binary>,
@@ -14,16 +14,16 @@ table! {
 table! {
     channel_messages (server, channel, message) {
         server -> Binary,
-        channel -> Integer,
-        message -> Integer,
+        channel -> BigInt,
+        message -> BigInt,
     }
 }
 
 table! {
     channels (server, id) {
         server -> Binary,
-        id -> Integer,
-        parent -> Nullable<Integer>,
+        id -> BigInt,
+        parent -> Nullable<BigInt>,
         name -> Text,
         icon -> Nullable<Integer>,
         deleted -> Bool,
@@ -34,7 +34,7 @@ table! {
     client_messages (server, client, message) {
         server -> Binary,
         client -> Binary,
-        message -> Integer,
+        message -> BigInt,
     }
 }
 
@@ -49,11 +49,11 @@ table! {
 
 table! {
     events (id) {
-        id -> Integer,
+        id -> BigInt,
         server -> Nullable<Binary>,
         invoker -> Nullable<Binary>,
-        channel1 -> Integer,
-        channel2 -> Integer,
+        channel1 -> BigInt,
+        channel2 -> BigInt,
         client -> Nullable<Binary>,
         typ -> Text,
         content -> Nullable<Binary>,
@@ -84,7 +84,7 @@ table! {
 table! {
     server_messages (server, message) {
         server -> Binary,
-        message -> Integer,
+        message -> BigInt,
     }
 }
 
